### DISCUSSION TITLE
chatwoot API integration options

### DISCUSSION THREAD

**Discussion by jcolinger3** [chatwoot/chatwoot#9301] [Self hosted]
New to this project, but it looks like something I'd like to use.
I am using Mattermost (basically Slack), 3x Zoom Team Chat, and 3x Incident IQ (help desk/ticket).
It is a challenge to manage all the channels.
I don't see any integration with Zoom Team Chat nor Incident IQ.  But I'm vaguely familiar with the APIs.

Is anybody self hosted and using an API method to connect chatwoot to Zoom Team Chat or Incident IQ... or really any other similar methods?

**Comment by bfontaine** [comment]
You problably don’t get any response because "Wondering about chatwoot" is too vague; you should use a proper title.

**Comment by sojan-official** [comment]
We have customers using the api channel to integrate various custom channels into Chatwoot. please refer https://www.chatwoot.com/hc/user-guide/articles/1677839703-how-to-create-an-api-channel-inbox
